Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -18,7 +19,19 @@ import { copyDir } from '../../utility/storage';
 import StorageService from '../../services/storage';
 import { IFileStorage } from '../storage/interface';
 import { BotProject } from '../bot/botProject';
-import { runYeomanTemplatePipeline } from '../../workers/templateInstallation.worker';
+import { templateGeneratorPath } from '../../settings/env';
+import { BackgroundProcessManager } from '../../services/backgroundProcessManager';
+
+let workerPath = '@bfc/server-workers/lib/templateInstallation.worker';
```

Comment:
preference to extract this into some utility and abstract the electron logic on L26. this sets up an easy extension in a hosted scenario where we may want some additional flexibility into where workers are held.

This would reduce to the following on L34
```typescript
const { runYeomanTemplatePipeline } = require(getWorkerPath('templateInstallation'));
```

Reply:
I would suggest going a step further and adding an abstraction in the `server-workers` package that hides this. 

```ts
import { ServerWorker } from '@bfc/server-workers';

// somewhere in code
await ServerWorker.enqueue('templateInstallation', { ...args });
```

Reply:
enqueue doesn't reflect the job being done, await on enqueue is confusing. A verb like executeJob or runJob is more appropriate imho. With enqueue, it feels like a job is ready to be dispatched synchronously (like in Android)